rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -1294,12 +1294,11 @@ RMW_Connext_Subscriber::create(
     sub_cft_params,
     &cft_topic);
 
-  if (RMW_RET_OK != cft_rc) {
-    if (RMW_RET_UNSUPPORTED != cft_rc) {
-      return nullptr;
-    }
-  } else {
+  if (RMW_RET_OK == cft_rc) {
     sub_topic = cft_topic;
+  } else {
+    // RMW_Connext_Subscriber needs this value to check if CFT is enabled or not.
+    sub_cft_expr = "";
```

Comment:
this is required for `RMW_Connext_Subscriber` w/o content_fitlered_topic, right?

Reply:
Yeah, RMW_Connext_Subscriber need this variable https://github.com/ros2/rmw_connextdds/blob/5d079a44112adce70451b26c6193b5de7fcbfad2/rmw_connextdds_common/src/common/rmw_impl.cpp#L1370, and set it into [cft_expression](https://github.com/ros2/rmw_connextdds/blob/5d079a44112adce70451b26c6193b5de7fcbfad2/rmw_connextdds_common/src/common/rmw_impl.cpp#L1148), which is used in https://github.com/ros2/rmw_connextdds/blob/5d079a44112adce70451b26c6193b5de7fcbfad2/rmw_connextdds_common/include/rmw_connextdds/rmw_impl.hpp#L490